services/QuillLMS/app/mailers/admin_diagnostic_reports/report_mailer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
+  class ReportMailer < ::UserMailer
+    default from: "The Quill Team <hello@quill.org>"
+
+    def csv_download_email(user_id, overview_url, skills_url, students_url)
+      user = User.find(user_id)
+      @addressee_name = user.first_name
+      @overview_url = overview_url
+      @skills_url = skills_url
+      @students_url = students_url
+      @human_date = DateTime.current.strftime('%B %d, %Y')
+
+      #email = user.email
+      email = 'thomas@quill.org'
```

Comment:
Better address this two lines before production :-)

Reply:
In the past, I've used an app setting, so that I can mail myself things during development, but also have the same code deployable to prod.

Reply:
I've followed your pattern.